content/docs/en/elements/components/slider.md

```diff
@@ -1,5 +1,37 @@
 ---
 title: Slider
+apiRef: https://docs.nativescript.org/api-reference/modules/_ui_slider_
```

Comment:
Should be https://docs.nativescript.org/api-reference/classes/_ui_slider_.slider